tools/parsergen/src/parsergen/python_direct_codegen.py

```
@@ -0,0 +1,1723 @@
+    owner: str
+    properties: tuple[str, ...]
+    class_name: str
+    stack_name: str
+    queue_name: str
+    field_names: tuple[str, ...]
+
+
+@dataclass(frozen=True, slots=True)
+class _LocalContinuationFinish:
+    number: int
+    layout: ContinuationLayout
```

Ruff (0.16.3)

[error] Undefined name `ContinuationLayout`

(F821)